MediaBrowser.Controller/Providers/IRemoteMetadataProvider.cs

```diff
@@ -8,20 +6,41 @@
 
 namespace MediaBrowser.Controller.Providers
 {
+    /// <summary>
+    /// Interface IRemoteMetadataProvider.
+    /// </summary>
     public interface IRemoteMetadataProvider : IMetadataProvider
     {
     }
 
+    /// <summary>
+    /// Interface IRemoteMetadataProvider.
+    /// </summary>
     public interface IRemoteMetadataProvider<TItemType, in TLookupInfoType> : IMetadataProvider<TItemType>, IRemoteMetadataProvider, IRemoteSearchProvider<TLookupInfoType>
         where TItemType : BaseItem, IHasLookupInfo<TLookupInfoType>
         where TLookupInfoType : ItemLookupInfo, new()
     {
+        /// <summary>
+        /// Gets the metadata for a specific LookupInfoType.
+        /// </summary>
+        /// <param name="info">The LookupInfoType to get metadata for.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/>.</param>
+        /// <returns>Task{MetadataResult{TItemType}}.</returns>
```

Comment:
This shouldn't just be the type, this should be an explanation of what gets returned.

Reply:
Still an issue